synonymdev/bitkit-react-native
# chore: remove ST v1

### Description

- remove v1 -> v2 migration code
- remove all v1 libs 
- remove ldk old restore
- remove ldk recover screen
- remove AuthWidget
- remove old env variables
- add error message if user tries to scan slashauth url

I will squash commits after review

### Linked Issues/Tasks


### Type of change

Refactoring

### Tests

No test

### QA Notes

Install prev version, add slashauth, upgrade, try to use slashauth. Widget should disapear
